h2o-r/h2o-package/R/models.R

```diff
@@ -3621,3 +3621,9 @@ h2o.getModelTree <- function(model, tree_number, tree_class = NA) {
 
 #' @export
 print.h2o.stackedEnsemble.summary <- function(x, ...) cat(x, sep = "\n")
+
+#' @export
+h2o.genericModel <- function(mojo_file_path){
+  file_key <- h2o.importFile(mojo_file_path, parse = FALSE)
+  h2o.generic(mojo_key = file_key)
```

Comment:
For some reason, `h2o.generic` call fails, as the function is unknown at the time this is called. In R local env, everything worked. 

When ran locally with `../../scripts/run.py --usecloud localhost:54321 --wipeall --geterrs --test testdir_algos/generic/runit_generic_model_mojo_gbm.R` it also fails:

```R
[2019-03-15 21:42:28] [ERROR] : Error: Test failed: 'Generic model from GBM MOJO'
Not expected: could not find function "h2o.generic"
1: withCallingHandlers(eval(code, new_test_environment), error = capture_calls, message = function(c) invokeRestart("muffleMessage"), 
       warning = function(c) invokeRestart("muffleWarning"))
2: eval(code, new_test_environment)
3: eval(code, new_test_environment)
4: withWarnings(test())
5: withCallingHandlers(expr, warning = wHandler)
6: test()
7: h2o.genericModel(mojo.original.path).
```